### PR TITLE
feat: create for-adoption page setup

### PR DESCRIPTION
**Pull request by AmandaKr**
## Jira issue

JIRA issue: [FSU-165](https://klau.atlassian.net/browse/FSU-165)

## Describe your changes

This PR introduces basic Next.js setup for a new "For adoption" page.
Link to the page:  [For adoption page](https://sterczaceuszy-git-fsu-165-for-adoption-setup-fsu.vercel.app/do-adopcji) 

## Checklist before requesting a review

- [x] Checked which branch you would merge.
- [x] Checked if the code follows the style guidelines of this project (use eslint and prettier).
- [x] Performed a self-review of code (bugs, clean code rules, etc).
- [x] Checked if added variables have unique names.
- [x] Checked if added code/function is not a duplicate. 
- [x] Tests have been added if possible.
- [x] New and existing unit tests pass locally with my changes.

### CODE REVIEW

**Review by paulgrym — APPROVED** [review]
The subpage exists under good url, there is a header and a footer so I think it all we need 😄

**Comment by Ebmili** [comment]
Well Done :)

**Review by Ebmili — APPROVED** [review]

File: packages/frontend/app/pages/do-adopcji.jsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+  const { data: adoptedAnimals } = await client.query({
+    query: GET_ADOPTED_ANIMALS,
+  })
+
+  return {
+    props: {
+      animals: animals.animalCollection,
+      adoptedAnimals: adoptedAnimals.animalCollection,
+      supporting: supporting.supportingCollection,
```
Comment (im-klau) [inline packages/frontend/app/pages/do-adopcji.jsx:37]: I guess this page only needs the `animals` data. You can get rid of the rest of them.

File: packages/frontend/app/templates/Adoption/Adoption.jsx [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+        <Typography variant="h4" as="h3">
+          🏡 {adoptedAnimals.total} Znalezionych domów
+        </Typography>
+        <Typography variant="h4" as="h3">
+          🏡 {animals.total + adoptedAnimals.total} Uratowanych zwierząt
+        </Typography>
+      </Container>
+    </Styled.Main>
+
+    <Footer />
```
Comment (im-klau) [inline packages/frontend/app/templates/Adoption/Adoption.jsx:51]: According to the Jira ticket, the contact banner should be included as well :D

**Review by im-klau — CHANGES_REQUESTED** [review]

File: packages/frontend/app/templates/Adoption/Adoption.jsx (hunk 2 above)
Comment (AmandaKr) [reply packages/frontend/app/templates/Adoption/Adoption.jsx:51]: ohh, right, I thought it was about the banner at the top :D

**Review by im-klau — APPROVED** [review]
🚀

**Pull request merged**